builder/virtualbox/ovf/config.go

```diff
@@ -87,15 +90,24 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	errs = packer.MultiErrorAppend(errs, c.VBoxManagePostConfig.Prepare(&c.ctx)...)
 	errs = packer.MultiErrorAppend(errs, c.VBoxVersionConfig.Prepare(&c.ctx)...)
 
-	if c.SourcePath == "" {
+	// Get downloadable URL
+	downloadableURL, err := common.DownloadableURL(c.SourcePath)
+	notURL := !strings.HasPrefix(downloadableURL, "http://") && !strings.HasPrefix(downloadableURL, "https://")
+
+	// SourcePath can't be empty and has to be 'downlodable'
+	if err != nil {
+		errs = packer.MultiErrorAppend(
+			errs, fmt.Errorf("source_path is invalid: %s", err))
+	} else if c.SourcePath == "" {
 		errs = packer.MultiErrorAppend(errs, fmt.Errorf("source_path is required"))
-	} else {
-		if _, err := os.Stat(c.SourcePath); err != nil {
-			errs = packer.MultiErrorAppend(errs,
-				fmt.Errorf("source_path is invalid: %s", err))
-		}
+	} else if _, err := os.Stat(c.SourcePath); err != nil && notURL {
+		errs = packer.MultiErrorAppend(
+			errs, fmt.Errorf("source_path is invalid: %s", err))
 	}
 
+	// Set SourcePath to downloadable URL value
+	c.SourcePath = downloadableURL
```

Comment:
since we're using `vm_path` rather than `SourcePath` in `StepImport`, I don't think we need to set this here.

We should probably also remove it as a field of `StepImport`